package.json

```diff
@@ -15,7 +15,7 @@
     "integration-test": "npm run build && npm run jest-integration-test",
     "jest-integration-test": "jest test/integration/",
     "lint:prettier": "prettier --check .",
-    "lint:eslint": "eslint . --ext .js,.ts",
+    "lint:eslint": "eslint . --ext .ts",
```

Comment:
Removing `js` from the linter is because we've moved the authorizer to typescript ?

Reply:
Yes. This was there to lint the original `authorizer.js` file. We don't want to lint it anymore, now it's the output of compiling the `.ts` file.